src/.prototype/musicutils/ts/musicutils.test.ts

```diff
@@ -55,13 +83,21 @@ describe('Music utilities', () => {
             expect(displayPitch('cb')).toBe('C♭');
         });
 
+        test('Pretty print cx and expect to be C𝄪', () => {
+            expect(displayPitch('cx')).toBe('C𝄪');
+        });
+
         test('Pretty print d# and expect to be D♯', () => {
             expect(displayPitch('d#')).toBe('D♯');
         });
 
         test('Pretty print dbb and expect to be D𝄫', () => {
             expect(displayPitch('dbb')).toBe('D𝄫');
         });
+
+        test('Pretty print sa and expect to be SA', () => {
```

Comment:
what is `sa`?

Reply:
Pitch of type `EAST_INDIAN_FLAT`. Innit?

Reply:
Alright.

Reply:
I don't think this needs to go in here.

Reply:
So should I just remove this test?

Reply:
Then would this be the right test for the case when no conditionals of `displayPitch` are applicable, and just the UpperCase version of the pitch argument is returned.

```
test('Pretty print c and expect to be C', () => {
            expect(displayPitch('c')).toBe('C');
});
```